pythontest.py

```diff
@@ -0,0 +1,267 @@
+import json
+import random
+import re
+import string
+from typing import List
+
+
+class Person:
+    name: str
+    age: int
+
+    def __init__(self, name=None, age=None):
+        self.name = name
+        self.age = age
+
+    def __str__(self):
+        return f"{self.name}({self.age})"
+
+    
+class Student(Person):
+    stt: int
+    score: float
+    
+    def toStringValue(self):
+        mystr = self.name + ' - ' + str(self.age) + ' - ' + str(self.score) + ' - ' + str(self.stt)
+        return mystr
+
+
+class Class:
+    total: int
+    students = []
+    grade: int
+    rank: int
+    name: str
+
+    def randomList(self):
+        if(self.total is not None):
+            for i in range(self.total):
+                stu = Student()
+                stu.name = 'student' + str(i)
+                stu.age = 18
+                stu.score = random.randint(1, 10)
+                stu.stt = i
+                self.students.append(stu)
+
+    def searchByName(self, keyword):
+            data = []
+            for i in range(self.total):
+                if(keyword  in self.students[i].name):
+                    data.append(self.students[i])
+                    # print(self.students[i])
```

Comment:
hàm này thiếu return kết qủa, có thể chạy vòng kiểu này để rút ngắn code/dễ đọc:
for student in self.students:
      if(keyword in student.name):
            data.append(student.name)